src/frontend/src/app/AppAuthenticated.tsx

```diff
@@ -103,7 +108,11 @@ const AppAuthenticated: React.FC<AppAuthenticatedProps> = ({ userId, userRole })
             <Route path={routes.CREDITS} component={Credits} />
             <Route path={routes.FINANCE} component={Finance} />
             <Route path={routes.CALENDAR} component={Calendar} />
-            <Route exact path={routes.HOME} component={Home} />
+            <Route
+              exact
+              path={routes.HOME}
+              render={() => <Home onMemberHomePage={onMemberHomepage} setOnMemberHomePage={setOnMemberHomePage} />}
```

Comment:
wait this isnt what we talked about with the global event listeners? we shouldnt be passing any params in here. the most we should be doing is setting query args but even that is a bit of a stretch. We should be using global listeneers to communciate between the Sidebar and the Home page